Library/Formula/mlkit.rb

```diff
@@ -0,0 +1,43 @@
+class Mlkit < Formula
+  desc "Compiler for the Standard ML programming language"
+  homepage "https://melsman.github.io/mlkit"
+  url "https://github.com/melsman/mlkit/archive/mlkit-4.3.9.tar.gz"
+  sha256 "3c6adbeb9a85f7b3586d0961fd3b170ff31e09fa0ff12889b76b9ceb459059c4"
+
+  depends_on "autoconf" => :build
+  depends_on "mlton" => :build
+  depends_on "gmp"
+
+  def install
+    system "./autobuild"
```

Comment:
Should this script just be calling autoreconf instead of what it currently does?